src/xhr.ts

```diff
@@ -287,6 +288,6 @@ export class XhrModule {
             throw new ApiResponseError('Unauthorized', response, responseBody);
         }
 
-        return new ApiResponse(response, responseBody);
+        return this.ajax(originalUrl, originalSettings);
```

Comment:
This could be better, but I am not sure: `return this.continueAfterTokenRequest(...)`. I will note to to gdjs refactor ticket.